src/CFamily/CMake/CompilationDatabaseLocator.cs

```diff
@@ -124,7 +124,9 @@ private string GetDefaultLocation(string rootDirectory, string activeConfigurati
                 return null;
             }
 
-            var evaluatedBuildRoot = macroEvaluationService.Evaluate(buildConfiguration.BuildRoot, buildConfiguration.Name, rootDirectory);
+            var evaluationContext = new EvaluationContext(buildConfiguration.Name, rootDirectory);
```

Comment:
I need to pass more information so I'll extend the context to include additional fields